### ISSUE TITLE
[Feature request] Grant all and Revoke all commands for basic skills

### ISSUE THREAD

**Issue by Cyo90** [impleri/player-skills#68]
This is another feature I think it would be great to have:

A "**grant all skills**" and "**revoke all skills**" **commands** just **for basic skills**. I know it's a strange petition but it would be a nice tool to have when creating and testing new quests/skills.

And it would be useful too in my case: I'm working on a modpack for friends where when you join the game you can choose between 2 modes: Adventure mode, where you have to do quests to gain skills which will unlock more recipes and items (from vanilla to mods, like game stages), and Free mode: unlock everything from the very start (just missing exclusive rewards from adventure mode). The trick is when the player select Free mode (quest from FTB Quests), the server will execute the command to grant the player all the basic skills (**set TRUE to all existing basic skills for this player**) and that's it. But right now I have to grant each skill one by one (and there is a TON).

Thank you again!

**Comment by impleri** [comment]
This can work for basic skills, but not for the others. It may be better to create a skill for freePlay and write the conditions for restrictions to fail if the player has freePlay. You could even create a wrapper function in KubeJS to ensure this always happens so you don't forget it. I have examples of that in the test-skills repo (server/constants)

**Comment by Cyo90** [comment]
Yes, the idea would be only for basic skills but your're right, your solution, at least for my case, is far better than a simple toggle. I would have more control over the skills I want to restrict on Free Mode.

Also, I looked for the test-skills repo but I'm not sure where to find it, could you provide me the link? Thank you.

**Comment by impleri** [comment]
Whoops, I didn't make it public. [It's now public](https://github.com/impleri/test-skills).
